CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Bit-Wallet-Manager
+
+### Introduction
+Thank for your contributing interest!! :tada:
+It's always good to contribute to open source projects to get more experience and together
+we improve the main ideia of this project. This is awesome for both sides :smile:
+So let's go the good part!
+
+### How to contribute
+
+Follow this easy steps right below and have fun!
+
+1. Take a look at the issue page
+2. Find the issue that fits you
+3. Comment on the issue, doing that the owner you'll see your requet to solve the problem
+4. Fork the repo to your account
+5. Switch to the `master` branch of the repo
+6. Initiate your issue solution
+7. Done? Great!! Just verify if you still on `master` branch then submit an PR for your issue, if get stuck go to(https://help.github.com/articles/creating-a-pull-request/))
+8. Done!! You did your part on this project! Yay! 
+9. Wait for the code review from your PR to get merged.
```

Comment:
What is merged is actually the PR, so the correct way to say would be:
`Wait for the code review and your PR to get merged.`